src/cascadia/TerminalControl/TermControl.h

```diff
@@ -75,6 +75,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         std::unique_ptr<::Microsoft::Console::Render::DxEngine> _renderEngine;
 
         Settings::IControlSettings _settings;
+        bool _focused;

```

Comment:
>bool _focused; [](start = 8, length = 14)

Could we use `_controlRoot.FocusState()`? [Click here](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.focusstate#Windows_UI_Xaml_Controls_Control_FocusState) for the doc

Reply:
So I did look into _controlRoot.FocusState(), and the value returned when inside MouseClick event handler is never FocusState.Unfocused. If I had to guess, _controlRoot's focus state is probably changed sometime between the GettingFocus event and the MouseClick event.

Reply:
Well that sucks. Thanks for checking.